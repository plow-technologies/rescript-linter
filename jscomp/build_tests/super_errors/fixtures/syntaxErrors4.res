
/* */
/* */
/* */
let a:int = "asdaaaaaaaaaaaaaaaaaaaaa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa
aa"
/* */
/* */
/* */
/* */
