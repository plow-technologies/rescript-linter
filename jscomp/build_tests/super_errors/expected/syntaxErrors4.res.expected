
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/syntaxErrors4.res[0m:[2m5:13-21:3[0m

   3 [2mâ”‚[0m /* */
   4 [2mâ”‚[0m /* */
   [1;31m5[0m [2mâ”‚[0m let a:int = [1;31m"asdaaaaaaaaaaaaaaaaaaaaa[0m
   [1;31m6[0m [2mâ”‚[0m [1;31maa[0m
   [2m.[0m [2mâ”‚[0m [2m...[0m
  [1;31m20[0m [2mâ”‚[0m [1;31maa[0m
  [1;31m21[0m [2mâ”‚[0m [1;31maa"[0m
  22 [2mâ”‚[0m /* */
  23 [2mâ”‚[0m /* */

  This has type: [1;31mstring[0m
  But it's expected to have type: [1;33mint[0m
  
  You can convert [1;33mstring[0m to [1;33mint[0m with [1;33mBelt.Int.fromString[0m.